src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/HomeStatisticsCellModel.swift

```diff
@@ -27,6 +27,14 @@ class HomeStatisticsCellModel {
 					.compactMap { cardID in
 						statistics.linkCards.first { $0.header.cardID == cardID }
 					}
+				#if DEBUG
+				if isUITesting {
+					var mockedPandemicLinkCardModel = SAP_Internal_Stats_LinkCard()
+					mockedPandemicLinkCardModel.header.cardID = 12
```

Comment:
You can use the existing mock `SAP_Internal_Stats_LinkCard.mock(cardID:updatedAt:url:)` from `SAP_Internal_Stats_LinkCard+Mock.swift`. You only have to set the check for UI test target :)